tests/pyret/tests/test-errors.arr

```diff
@@ -125,6 +125,14 @@ check:
     e-non-ctor-app2.loc satisfies S.is-srcloc
   end
 
+  some-method = method(x): x end
+  e-method-app = get-err(lam(): some-method() end)
+  e-method-app satisfies E.is-non-function-app
+  when E.is-non-function-app(e-method-app) block:
+    tostring(e-method-app.non-fun-val) is "<function>"
```

Comment:
This is just annoying :(  "It's a non-function application.  The non-function thing was <function>."  Wha?  I know you can't do better, because dot-access automatically curries the method over its receiver, but this is still annoying...